src/containerapp/azext_containerapp/containerapp_job_decorator.py

```diff
@@ -278,7 +276,7 @@ def construct_payload(self):
         eventTriggerConfig_def = None
         if self.get_argument_trigger_type is not None and self.get_argument_trigger_type().lower() == "event":
```

Comment:
Should we remove `if self.get_argument_trigger_type() is not None` As `Usage error: --trigger-type is required`?